Mathlib/Analysis/NormedSpace/OperatorNorm/Prod.lean

```diff
@@ -139,3 +151,38 @@ end Prod
 end ContinuousLinearMap
 
 end SemiNormed
+
+section Normed
+
+variable [NormedAddCommGroup E] [NormedSpace 𝕜 E] [NormedAddCommGroup F] [NormedSpace 𝕜 F]
+  [NormedAddCommGroup G] [NormedSpace 𝕜 G]
```

Comment:
I think that you only need `E` to be normed for `norm_fst` and only `F` to be normed for `norm_snd`. Also, you don't need `G` at all.